charts/stable/fireflyiii/values.yaml

```diff
@@ -34,6 +34,7 @@ env:
   DB_CONNECTION: pgsql
   DB_PORT: 5432
   APP_KEY: AGcfkCUS233ZWmBXztYbdyCs2u7kkz55
+  STATIC_CRON_TOKEN: "AGcfkCUS233ZWmBXztYbdyCs2u7kkz55"
```

Comment:
remove this and add the same secretKeyRef like below

Reply:
Should be okay now!